browser/src/geo/crs/CRS.js

```diff
@@ -38,6 +38,16 @@ L.CRS = {
 		return Math.pow(1.2, zoom);
 	},
 
+	// equivalent to doing an unproject with oldZoom then a project with newZoom
+	// except that unproject is technically invalid (so possibly confusing) for any non-css-pixel
+	// but this function will work with any scaling (including twips or core pixels)
+	rescale: function (point, oldZoom, newZoom) {
+		return L.point(
+			point.x * this.scale(newZoom - oldZoom),
```

Comment:
I will check the commit in detail now but, do we need to add a `* 0.5` here?

Reply:
I don't... believe so? I got to this point by taking the project(unproject()) functions, writing down what they did and cancelling terms... it's possible I missed something but I'd find it weird if it were a factor of 0.5

Reply:
I also checked this locally in a repl to see that it outputted the same thing ... I would write a unit test but as far as I know we don't have anything like [vitest](https://vitest.dev/) to test pure JavaScript so I'd have to write something in cypress ... maybe that's a thing for next hack week